mantle/platform/platform.go

```diff
@@ -466,6 +466,8 @@ func CheckMachine(ctx context.Context, m Machine) error {
 		distribution = "fcos"
 	case `rhcos-`:
 		distribution = "rhcos"
+	case `centos-`:
+		distribution = "rhcos"
```

Comment:
Minor Go nit, this can be:

```suggestion
        case `rhcos-`, `centos-`:
		distribution = "rhcos"
```